spec/javascript/packs/pw-strength-spec.js

```diff
@@ -36,34 +36,43 @@ describe('pw-strength', () => {
 
   describe('getFeedback', () => {
     const EMPTY_RESULT = '&nbsp;';
+    const MINIMUM_LENGTH = 12;
+    const FORBIDDEN_PASSWORDS = '["gsa", "Login.gov"]';
```

Comment:
Forbidden passwords is passed as an array argument. This just happens to work also since `String#includes` would check for the same presence somewhere in the JSON array.

Ideally we'd have TypeScript enforcing our types for these files.

```suggestion
    const FORBIDDEN_PASSWORDS = ['gsa', 'Login.gov'];
```